trillian/integration/chains.go

```diff
@@ -0,0 +1,245 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
2018 or did it move? Hard to tell on Github.

Reply:
This one moved; `copier.go` is the new stuff (and has © 2018)